### PR TITLE
Reopen files that were open #20

### PR DESCRIPTION
**Pull request by Perkovec**
[Reopen files that were open](https://github.com/egoist/eme/issues/20)
ide

### CODE REVIEW

File: src/utils/config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Configs, used in renderer process
+'use strict'
+const Config = require('electron-config')
```
Comment (egoist) [inline src/utils/config.js]: `electron-config` is not installed in https://github.com/egoist/eme/blob/dev/package.json , it's installed as `./app`'s [dependency](https://github.com/egoist/eme/blob/dev/app/package.json),  while in fact we can use pass the `config` object from the main process to the render process, just like how we set the `$state` [here](https://github.com/egoist/eme/blob/dev/app/package.json)

File: app/eme/window.js [hunk 2]
```diff
@@ -4,9 +4,17 @@ const {
   BrowserWindow,
   shell
 } = require('electron')
+const Config = require('electron-config')
```
Comment (egoist) [inline app/eme/window.js]: why not use https://github.com/egoist/eme/blob/dev/app/eme/config.js directly?

```js
const config = require('./config')
```

File: app/eme/window.js [hunk 3]
```diff
@@ -57,6 +65,7 @@ class Window {
     win.$state = {
       unsaved: 0
     }
+    win.configs = config
```
Comment (egoist) [inline app/eme/window.js]: I guess all these shared data should be named `$xxx`, eg: `win.$config = config`

File: app/eme/window.js (hunk 2 above)
Comment (Perkovec) [inline app/eme/window.js]: separate file for the renderer process configs?

Comment (egoist) [inline app/eme/window.js]: since it's app-level state, seems no need to separate file

**Comment by Perkovec** [comment]
@egoist fixed

**Comment by egoist** [comment]
Ah, it feels quite annoying to see the previous files when I open the app. People didn't use a markdown editor as often as they use code editor, every time they open the editor they most likely want to write about some fresh new ideas, that's why they may feel annoyed seeing the previous files.

But this is just my own opinion after trying this feature out. 😂 Still thanks for your work

**Comment by Perkovec** [comment]
It would be necessary to make the settings window that you can choose to open recent files or not

**Comment by Perkovec** [comment]
@egoist completed

**Pull request merged**